components/PanelGroup/PanelGroup.jsx

```diff
@@ -48,7 +48,9 @@ const Panel = ({ children, collapsed, icon, name, subtitle, title, toggleIconNam
           </BootstrapPanel.Title>
 
           <If condition={notification}>
-            {notification}
+            <span className="panel-heading-notification">
+              {notification}
+            </span>
```

Comment:
We have to ensure consistent spacing here so that the actual rotation doesn't look dorky (margin on wrong side)